testutil/client.go

```diff
@@ -223,6 +223,21 @@ func RetryQuery(dg *dgo.Dgraph, q string) (*api.Response, error) {
 	}
 }
 
+func RetryAlterSchema(dg *dgo.Dgraph, op *api.Operation) error {
```

Comment:
Shouldn't we rename it to `RetryAlter` instead? As this function is used for `drop all` operation as well.